pythoncalculator/multiply.py

```diff
@@ -0,0 +1,2 @@
+def multiply(x, y):
+    return x + y
```

Comment:
This line seems to be adding rather than multiplying
```suggestion
    return x * y
```